src/avram/credentials.cr

```diff
@@ -80,6 +80,17 @@ class Avram::Credentials
     @query.try(&.strip).presence
   end
 
+  # This is the full connection string used
+  # to connect to the PostgreSQL server.
+  def connection_string : String
+    String.build do |io|
```

Comment:
In order to run SQL before a database exists, we need the connection string `postgres://user:pass@host:5432`

Reply:
Oh, I was just saying that I moved this logic out so I can differentiate between

```
postgres://user:pass@host:5432
```

and

```
postgres://user:pass@host:5432/some_db
```